src/shared/components/common/pictrs-image.tsx

```diff
@@ -20,13 +21,35 @@ interface PictrsImageProps {
   cardTop?: boolean;
 }
 
-export class PictrsImage extends Component<PictrsImageProps, any> {
+interface PictrsImageState {
+  src: string;
+}
+
+function handleImgLoadError(i: PictrsImage) {
+  i.setState({
+    src: `${getStaticDir()}/assets/images/broken-image-fallback.png`,
```

Comment:
Try [this solution](https://stackoverflow.com/a/13118956/1655478) to see if it works.

`<img src="foo.jpg" onerror="this.src='`NEW_SRC_LOC`>`

Then we could avoid having to have to create a src state variable, and hooks, just to handle the error case.

Reply:
> `<img src="foo.jpg" onerror="this.src='`NEW_SRC_LOC`>`

I tried something similar at some point. Worked fine in dev. But in prod the CSP complained about inline-scripts.

Reply:
Ah, in that case this is good then.